src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -82,6 +82,33 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
+{{ if OryxAppInsightInjectCommand | IsNotBlank }}
+echo
+echo "Preparing Envrionment to set up application insight..."
+echo "Running '{{ OryxAppInsightInjectCommand }}'..."
+echo
+{{ OryxAppInsightInjectCommand }}
+echo
+echo "Creating application insight loader file to '$DESTINATION_DIR'"
+echo "// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+var appInsights = require('applicationinsights');
```

Comment:
utilize `NodeAppInsightPackageName`

Reply:
what do you mean? if you are asking different variable name I followed npm site and there it's been used like this https://www.npmjs.com/package/applicationinsights